### PR TITLE
Backend debug

### PR DESCRIPTION
**Pull request by boulatrousse**
Debug fixs for the backend app `users`.

### CODE REVIEW

File: frontend/js/modules/friendList/friendsWs.js
Comment (gclement42) [inline frontend/js/modules/friendList/friendsWs.js:1]: In this file you have a lot of send function, I think you can group them

File: frontend/js/modules/friendList/friendsWs.js [hunk 1]
```diff
@@ -143,4 +137,18 @@ function closeWs () {
 	}
 }
 
+function checkWs() {
```
Comment (gclement42) [inline frontend/js/modules/friendList/friendsWs.js:140]: I already have a function for check if a webSocket is open, you can use it

File: py_backend/users/models.py [hunk 2]
```diff
@@ -36,7 +36,9 @@ class Meta:
 	friends = models.ManyToManyField("self", blank=True)
 	is_42auth = models.BooleanField(default=False)
 	is_online = models.BooleanField(default=False)
+	is_logged_in = models.BooleanField(default=False)
```
Comment (gclement42) [inline py_backend/users/models.py:39]: What are the is_logged_in and sessionkey fields used for?

**Review by maujogue — APPROVED** [review]
Good protection overall, it has the problem caused by the previous PR ( can't modify info because of 'already connected somewhere else'), but this is fixed in my next one.

**Review by maujogue — APPROVED** [review]

**Pull request merged**